phoenix-core-server/src/main/java/org/apache/phoenix/coprocessor/CDCGlobalIndexRegionScanner.java

```diff
@@ -257,7 +252,7 @@ private Object getColumnValue(Cell cell, PDataType dataType) {
 
     private Object getColumnValue(byte[] cellValue, int offset, int length, PDataType dataType) {
         Object value;
-        if (dataType.getSqlType() == Types.BINARY) {
+        if (CDCUtil.isBinaryType(dataType)) {
```

Comment:
This is an improvement with this PR, correct?

Reply:
This is just a utility method to abstract out the 3 types that we are considering as binary type.

Reply:
Oh i missed that it was Types.BINARY which anyways apply to them, i think i misunderstood earlier that we only had Binary datatype, and not VARBINARY.